src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Deprecate using `DateTime` or `DateTimeImmutable` model data with a different timezone than configured with the
    `model_timezone` option in `DateType`, `DateTimeType`, and `TimeType`
+ * Deprecate `PostSetDataEvent::setData()` and `PostSubmitEvent::setData()`
```

Comment:
* Deprecate `PostSetDataEvent::setData()` and `PostSubmitEvent::setData()`, these methods will throw an exception starting with 7.0